File: src/Snake.java
public class Snake extends Animal {
    private final String _snakeName;

    public Snake(String snakeName, String animalSpecies, short numberOfLegs, short animalAge) {
        super(animalSpecies, numberOfLegs, animalAge);
        _snakeName = snakeName;
    }

    @Override
    public void MakeSound() {
        System.out.printf("%s called %s makes sound <<Hiss>>!%n", _animalSpecies, _snakeName);
    }
}

File: src/Snake.cs
using Proiect_Animal_Sounds.src.Interfaces;

namespace Proiect_Animal_Sounds.src
{
    internal class Snake : IAnimal
    {
        public void MakeSound()
        {
            Console.WriteLine($"<{GetType().Name}> makes sound: <Hiss>");
        }
    }
}
